Makefile

```
@@ -3,6 +3,10 @@
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
+# best if we could detect this. If we cannot, we need to document it somewhere.
+# then we can add a patch in the `PHONY: bundle`
+
+PREVIOUS_VERSION ?= 0.0.1
 VERSION ?= 0.0.2
```

Ran `make bundle -e PREVIOUS_VERSION=0.0.3 -e VERSION=0.0.4`. No error.
To make the next release easier, should we preset the envs to:
```suggestion
PREVIOUS_VERSION ?= 0.0.3
VERSION ?= 0.0.4
```

---

I think `VERSION ?= 0.0.0-dev` should be ok. `PREVIOUS_VERSION` may need to be an existing bundle release judging from some examples [here](https://docs.openshift.com/container-platform/4.4/operators/understanding_olm/olm-understanding-olm.html#olm-upgrades_olm-understanding-olm) which implies the `replaces` is an version from an update graph which maybe built from  existing CSVs.